models/api/cohorts.js

```diff
@@ -7,6 +7,7 @@ const CohortSchema = new Schema({
         required: true,
```

Comment:
I'm still not sure I understand the need for the `startingMonth` and `year` properties on this schema if both of those things are also part of the `startingDate` and `endingDate`. Unless there's a good reason, I'd vote to remove `startingMonth` and `year` from this schema.